### PR TITLE
DOC-734 Minor updates in v4.39.0 release

### PR DESCRIPTION
**Pull request by asimms41**
## Description

Resolves [DOC-734](https://redpandadata.atlassian.net/browse/DOC-734)
Review deadline: 14 November

Covers minor changes in v4.39.0 release

## Page previews

- [`mqtt` input](https://deploy-preview-96--redpanda-connect.netlify.app/redpanda-connect/components/inputs/mqtt/#urls) and [output](https://deploy-preview-96--redpanda-connect.netlify.app/redpanda-connect/components/outputs/mqtt/#urls) - improved `urls` field description
- [`aws_sqs` output](https://deploy-preview-96--redpanda-connect.netlify.app/redpanda-connect/components/outputs/aws_sqs/) - `url` field now supports interpolation functions
- [Telemetry configuration guide](https://deploy-preview-96--redpanda-connect.netlify.app/redpanda-connect/configuration/telemetry/#disabling-the-telemetry-service): disable telemetry using rpk command

## Checks

- [x] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

[DOC-734]: https://redpandadata.atlassian.net/browse/DOC-734?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by rockwotj — APPROVED** [review]

File: modules/components/pages/inputs/mqtt.adoc [hunk 1]
```diff
@@ -89,8 +89,13 @@ You can access these metadata fields using xref:configuration:interpolation.adoc
...
 === `urls`
 
-A list of URLs to connect to. If an item of the list contains commas it will be expanded into multiple URLs.
+A list of URLs to connect to. Use the format `scheme://host:port`, where:
 
+* `scheme` is one of the following: `tcp`, `ssl`, `ws`
+* `host` is the IP address or hostname
+* `port` is the port on which the MQTT broker accepts connections
+
+If an item of the list contains commas, it is expanded into multiple URLs.
```
Comment (JakeSCahill) [inline modules/components/pages/inputs/mqtt.adoc:98]:
```suggestion
If an item in the list contains commas, it is expanded into multiple URLs.
```

File: modules/configuration/pages/telemetry.adoc [hunk 2]
```diff
@@ -43,10 +43,16 @@ The code responsible for extracting usage data is available in the `connect` rep
 
 To avoid analyzing instances used for experimentation or testing, a Redpanda Connect instance must have been running for at least 5 minutes before any usage data is collected. Once usage data starts to be emitted, it is sent to the collection server once every 24 hours. 
 
-== Blocking the telemetry service
+== Disabling the telemetry service
```
Comment (JakeSCahill) [inline modules/configuration/pages/telemetry.adoc:46]:
```suggestion
== Disable the telemetry service
```

File: modules/configuration/pages/telemetry.adoc [hunk 3]
```diff
@@ -43,10 +43,16 @@ The code responsible for extracting usage data is available in the `connect` rep
...
 
 The telemetry service configuration is only included in the build artifacts released through https://github.com/redpanda-data/connect/releases[GitHub] or Redpanda’s official https://hub.docker.com/r/redpandadata/connect/[Docker images]. Custom builds do not send usage data.
```
Comment (JakeSCahill) [inline modules/configuration/pages/telemetry.adoc:48]: We should clarify that this includes the build artifacts `rpk` downloads when you run `rpk connect run`

File: modules/configuration/pages/telemetry.adoc [hunk 4]
```diff
@@ -43,10 +43,16 @@ The code responsible for extracting usage data is available in the `connect` rep
...
 The telemetry service configuration is only included in the build artifacts released through https://github.com/redpanda-data/connect/releases[GitHub] or Redpanda’s official https://hub.docker.com/r/redpandadata/connect/[Docker images]. Custom builds do not send usage data. 
 
-To continue to use the official Redpanda artifacts or images but disable the telemetry service, you can block internet traffic. Redpanda Connect continues to operate normally if it is unable to send usage data.
+To continue to use the official Redpanda artifacts or images but disable the telemetry service, you can either block internet traffic or run the following command:
+
+```bash
+rpk connect run --disable-telemetry
+```
+
+Redpanda Connect will continue to operate normally.
```
Comment (JakeSCahill) [inline modules/configuration/pages/telemetry.adoc:56]:
```suggestion
Disabling telemetry does not affect the normal operation of Redpanda Connect.


```

**Review by JakeSCahill — APPROVED** [review]
Minor suggestions

File: modules/components/pages/inputs/mqtt.adoc (hunk 1 above)
Comment (asimms41) [reply modules/components/pages/inputs/mqtt.adoc:98]: Updated.

File: modules/configuration/pages/telemetry.adoc (hunk 3 above)
Comment (asimms41) [reply modules/configuration/pages/telemetry.adoc:48]: Updated.

File: modules/configuration/pages/telemetry.adoc (hunk 4 above)
Comment (asimms41) [reply modules/configuration/pages/telemetry.adoc:56]: Updated.

**Pull request merged**